test/yaml-mode-test.el

```diff
@@ -0,0 +1,117 @@
+;;;; yaml-mode-test.el --- Tests for yaml-mode  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2010-2022 Yoshiki Kurihara
```

Comment:
This is incorrect, you wrote the file, so you have the copyright and authorship.